app/graph/mutations/api_key_mutations.rb

```diff
@@ -0,0 +1,36 @@
+module ApiKeyMutations
+  MUTATION_TARGET = 'api_key'.freeze
+  PARENTS = ['team'].freeze
+
+  module CreateFields
+    extend ActiveSupport::Concern
+
+    included do
+      argument :title, GraphQL::Types::String, required: false
+      argument :description, GraphQL::Types::String, required: false
+    end
+  end
+
+  class Create < Mutations::CreateMutation
+    include CreateFields
+
+    def resolve(**inputs)
+      begin
+        bot_user = BotUser.new
+        bot_user.name = inputs[:title]
+        bot_user.login = inputs[:title]
+        bot_user.save!
+
+        bot_user.api_key.title = inputs[:title]
+        bot_user.api_key.description = inputs[:description]
+        bot_user.api_key.save!
+
+        { success: true, api_key: bot_user.api_key }
+      rescue
+        { success: false, error: 'Could not create API key' }
+      end
+    end
```

Comment:
I'm curious, Jay: Why you had to use a custom `resolve` method? The `Mutations::CreateMutation` base class you're using should take care of everything. If this is because you need the `BotUser` instance to be created, I suggest you handle it at the model level as a `before_create` callback.